providers/tentacle.rb

```diff
@@ -168,7 +169,11 @@
     notifies :restart, "windows_service[#{service_name}]", :delayed
   end
 
-  new_resource.updated_by_last_action(actions_updated?([register_instance]))
+  service = windows_service service_name do
```

Comment:
Yup that is correct.  That service resource does not need to be included as it is dependent on the tentacle.  If it was also performing an action by default I would include it but in this case it is only being notified by the existing resource.